packages/smart-wallet/src/offers.js

```diff
@@ -111,7 +116,6 @@ export const makeOfferExecutor = ({
           offerArgs,
         );
         logger.info(id, 'seated');
-        updateStatus({});
```

Comment:
I included "exit offer" in this PR because it influenced the design here. We return the seat if this succeeds rather than updating the status with `{}`.

I can take this commit out of this PR though if needed.